docs/narr/project.rst

```diff
@@ -316,12 +317,13 @@ the case, if you use a browser running on the same system as Pyramid, it will
 be able to access the application via ``http://127.0.0.1:6543/`` as well as via
 ``http://192.168.1.50:6543/``. However, *other people* on other computers on
 the same network will also be able to visit your Pyramid application in their
-browser by visiting ``http://192.168.1.50:6543/``.
+browser by visiting ``http://192.168.1.50:6543/``. Same holds true if you use
+ipv6. ``[::]`` means the same as ``0.0.0.0`` but for ipv6 protocol.
```

Comment:
More minor grammar fixes.

IPv6. ``[::]`` means the same as ``0.0.0.0``, but for IPv6.